server.js

```diff
@@ -1,24 +1,50 @@
 var path        = require('path'),
     fs          = require('fs'),
     merge       = require('merge'),
+    session     = require('express-session'),
+    cookieParser= require('cookie-parser'),
+    flash       = require('connect-flash'),
     express     = require('express'),
     browserSync = require('browser-sync'),
     nunjucks    = require('express-nunjucks'),
     _           = require('underscore'),
     routes      = require(__dirname + '/app/routes.js'),
     dis_routes  = require(__dirname + '/app/views/display/routes.js'),
+    authRoutes  = require(__dirname + '/app/authRoutes.js'),
+    passport    = require('passport'),
     favicon     = require('serve-favicon'),
     app         = express(),
-    port        = process.env.PORT || 3100,
+    port        = process.env.PORT || 8090,
     env         = process.env.NODE_ENV || 'development';
 
+require(__dirname + '/app/passport.js'),
+
 function requireHTTPS(req, res, next) {
   // Heroku terminates SSL connections at the load balancer level, so req.secure will never be true
   if (req.headers["x-forwarded-proto"] != "https") {
     return res.redirect('https://' + req.hostname + req.url);
   }
   next();
 }
+
+if(!process.env.COOKIE_SECRET) {
+  console.warn('COOKIE_SECRET is not set. Unsafe cookie secret will be used instead.');
```

Comment:
perhaps we could use the bunyan logger in all new/edited files? Would be awesome if the logger creation was externalised to a separate file so we don't define logging levels in each file.